lib/config/migrations/custom/bb-use-default-reviewers-migration.ts

```diff
@@ -0,0 +1,17 @@
+import is from '@sindresorhus/is';
+import { AbstractMigration } from '../base/abstract-migration';
+
+export class BbUseDefaultReviewersMigration extends AbstractMigration {
+  override readonly propertyName = 'bbUseDefaultReviewers';
+
+  override run(value: unknown, key: string, parentKey?: string): void {
+    const platformOptions = this.get('platformOptions') ?? {};
+    if (is.boolean(value) && parentKey !== 'platformOptions') {
+      this.delete();
+      this.setHard('platformOptions', {
```

Comment:
wouldn't thise move the `platformOptions` key if already exist in config?

Reply:
No, it will update it only.

Reply:
yes, but since its a new config option, that should not happen as long as we migrated all the options in one pr.